plugins/talk-plugin-featured-comments/client/containers/TabPane.js

```diff
@@ -102,6 +102,7 @@ const enhance = compose(
             tags: ["FEATURED"]
             sortOrder: $sortOrder
             sortBy: $sortBy
+            excludeIgnored: true
```

Comment:
Should be `excludeIgnored: $excludeIgnored`, taking the variable from the query directly, which is set to false when user is not logged in otherwise true.

Reply:
Got it. I tried that and it didn't work. I will check again. One sec.

Reply:
Perfect 👍

Reply:
Thanks for the clarification